### ISSUE TITLE
Alpine linux support

### ISSUE THREAD

**Issue by ohardy** [mac-s-g/react-json-view#313]
Hi,

I discovered that docusaurus uses this package.

Indeed, since this morning, I can't do any more `npm install` in my alpine docker images because of the dependency with the npm `node` package.

```shell
npm-why node
> @docusaurus/preset-classic > @docusaurus/plugin-debug > react-json-view > node@10.23.0
```

And node npm package doesn't support alpine for now (https://github.com/aredridel/node-bin-gen/issues/48)

So I create this issue to know if it's easy for you to remove usage of this package, or if you know a solution for this problem ?

The error I have on npm install:
```shell
npm ERR! code 127
npm ERR! path /src/node_modules/core-js-pure
npm ERR! command failed
npm ERR! command sh -c node -e "try{require('./postinstall')}catch(e){}"
npm ERR! Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /src/node_modules/.bin/node)
npm ERR! Error relocating /src/node_modules/.bin/node: gnu_get_libc_version: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: __register_atfork: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: __strdup: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: setcontext: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: makecontext: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: backtrace: symbol not found
npm ERR! Error relocating /src/node_modules/.bin/node: getcontext: symbol not found
```

And testing directly in the docker image:
```shell
/src/node_modules/.bin # ./node 
Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by ./node)
Error relocating ./node: gnu_get_libc_version: symbol not found
Error relocating ./node: __register_atfork: symbol not found
Error relocating ./node: __strdup: symbol not found
Error relocating ./node: setcontext: symbol not found
Error relocating ./node: makecontext: symbol not found
Error relocating ./node: backtrace: symbol not found
Error relocating ./node: getcontext: symbol not found
```

Thanks

**Comment by mac-s-g** [comment]
yes, i think i can resolve this today.
thanks for reporting.

**Comment by mac-s-g** [comment]
as soon as ci finished, i'll merge and deploye #314

**Comment by mac-s-g** [comment]
@ohardy i removed the peer dependency on node, merged and published to npm at v `1.20.1`.

can you let me know if that resolves the issues for you?

thanks again for all the detail you included with this issue.

**Comment by ohardy** [comment]
@mac-s-g Thank you so much, everything is ok now :)
